samples/repo-search/shared/gradle.properties

```diff
@@ -0,0 +1 @@
+org.jetbrains.compose.experimental.uikit.enabled=true
```

Comment:
Yes, because I have moved some of the common Composables UI in the demo app under the common source in the `shared module`. So you will get build failures trying to build the android or desktop demo app if you don't explicitly opt-in.

Reply:
Ah fair, I didn't see this was in the sample app tbh. All's good